src/platform/components/Header/index.tsx

```diff
@@ -8,20 +8,41 @@ import Logo from "assets/img/logo.png";
 
 type HeaderProps = {
   backButtonEnabled?: boolean;
+  navsEnabled?: boolean;
 };
 
-const Header: FC<HeaderProps> = ({ backButtonEnabled }) => {
+const Header: FC<HeaderProps> = ({
+  backButtonEnabled = false,
+  navsEnabled = false,
+}) => {
   const history = useHistory();
 
   return (
-    <header className={styles.header}>
+    <header
+      className={styles.header}
+      style={{
+        justifyContent: backButtonEnabled
+          ? "space-between"
+          : navsEnabled
+          ? "space-between"
+          : "flex-start",
+      }}
```

Comment:
No need to set it like so, if you set margin-inline-end: auto; for the logo container

Reply:
I explained it to you in the previous comment. This is the best solution in this case

Reply:
Please remove it & set margin-inline-end for the logo to "auto"

Reply:
Done